app/views/admin/tags/edit.html.erb

```diff
@@ -18,8 +18,8 @@
       <% if @tag_moderators.exists? %>
         <ul class="list-none">
           <% @tag_moderators.each do |user| %>
-              <li class="flex justify-between">
-                <a href="/<%= URL.user(user) %>">@<%= user.username %></a>
+              <li class="flex justify-between mb-2">
+                <a href="/<%= URL.user(user) %>" target="_blank" rel="noopener noreferer">@<%= user.username %></a>
```

Comment:
just noticed this link is broken, it should be without the leading `/` in front of `<%= URL.user(user) %>`

Reply:
Great catch thanks! Done